test/test_serializers.py

```diff
@@ -0,0 +1,62 @@
+"""
+
+    test_parse.py
+
+    Tests for Greynir module
+
+    Copyright(C) 2019 by Miðeind ehf.
+    Original author: Vilhjálmur Þorsteinsson
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+"""
+
+import json
+
+import pytest
+
+
+@pytest.fixture(scope="module")
+def r():
+    """ Provide a module-scoped Greynir instance as a test fixture """
+    from reynir import Greynir
+    r = Greynir()
+    yield r
+    # Do teardown here
+    r.__class__.cleanup()
+
+
+def test_serializers(r):
```

Comment:
Add test here with more token types (dates, times, numbers, amounts...)

Reply:
...and person names, they might trip up the serialization.

Reply:
I updated the text to include the token types you mentioned. But perhaps it would be a good idea to have a standard list of texts somewhere in the project that can be imported and used in test cases like these. What do you think?